### ISSUE TITLE
Plugin repo review master thread

### ISSUE THREAD

**Issue by stuwest** [Automattic/wp-cldr#86]
Submitted for review this afternoon let's use this thread to track.

<img width="605" alt="screen shot 2016-03-17 at 4 22 37 pm" src="https://cloud.githubusercontent.com/assets/3966960/13865372/f8d9bfc8-ec67-11e5-9cd7-b55e1034f2f7.png">

**Comment by stuwest** [comment]
:boom: 

> From: "WordPress.org" <plugins@wordpress.org>
> Subject: [WordPress.org Plugins] Request Approved: wp-cldr
> Date: March 18, 2016 at 1:01:37 PM PDT
> To: stu@automattic.com
> 
> stuwest,
> 
> Your plugin hosting request has been approved.
> 
> Within one hour, you will have access to your SVN repository at
> 
> https://plugins.svn.wordpress.org/wp-cldr/
> 
> with your WordPress.org username and password (the same one you use on the forums).
> 
> Here's some handy links to help you get started.
> 
> Using Subversion with the WordPress Plugin Directory
> https://wordpress.org/plugins/about/svn/
> 
> FAQ about the WordPress Plugin Directory
> https://wordpress.org/plugins/about/faq/
> 
> WordPress Plugin Directory readme.txt standard
> https://wordpress.org/plugins/about/readme.txt
> 
> readme.txt validator:
> https://wordpress.org/plugins/about/validator/
> 
> Enjoy!

**Comment by stuwest** [comment]
@jblz, you have any scripts/tooling you like for getting releases from Github into the WP plugin repo?

**Comment by jblz** [comment]
I don't have a lot of experience going between svn & git in any automated fashion, so I asked around (props @daggerhart).

These seem like good resources:
* https://gist.github.com/kasparsd/3749872 (note the update at the top that points to [this repo of bash scripts](https://github.com/kasparsd/wp-deploy) instead
* http://jeremypry.com/wordpress-plugin-development-with-github/

**Comment by stuwest** [comment]
Thanks. Super useful. I went with the manual way this time following the [directions sent in the plugin repo acceptance email](https://wordpress.org/plugins/about/svn/). Seemed to work fine and plugin is live now. I left it at 0.9 so we have a record of what we actually submitted. Will make a few tweaks to the documentation and updated it to 1.0.

Some screenshots for posterity.

![screen shot 2016-03-21 at 11 49 42 am](https://cloud.githubusercontent.com/assets/3966960/13930349/f7370f8a-ef5b-11e5-9929-f1fb3dcd0f30.png)
 
![screen shot 2016-03-21 at 11 46 20 am](https://cloud.githubusercontent.com/assets/3966960/13930358/00771f54-ef5c-11e5-8d92-b8d5c3af26b9.png)

![screen shot 2016-03-21 at 11 46 52 am](https://cloud.githubusercontent.com/assets/3966960/13930367/0691ce8e-ef5c-11e5-8cd2-b69e9cac0327.png)
